server/routes/next-steps.js

```diff
@@ -12,17 +15,25 @@ module.exports = [
     options: {
       description: 'Results Page',
       handler: async (request, h) => {
+        let payload
         const { polygon, encodedPolygon } = checkParamsForPolygon(request.query)
         const [contactData, floodData, { isRiskAdminArea: isRiskAdmin }] = await Promise.all([
           request.server.methods.getPsoContactsByPolygon(polygon),
           request.server.methods.getFloodZoneByPolygon(polygon),
           isRiskAdminArea(polygon)]
         )
-
+        try {
```

Comment:
This is repeated across routes.

Could you make a separate method getProductOnePause, that wraps all the error checks and just returns pauseP1Data.

so each route will just need a const pauseP1Data = await getProductOnePause() line